src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -115,12 +116,19 @@ public void testToString() {
     }
 
     @Test
-    public void testKeyEncryption() {
+    public void testKeyEncryption() throws InvalidParametersException {
         String msg = "Test decryption";
         String decrptedMsg;
 
         decrptedMsg = StringHelper.decrypt(StringHelper.encrypt(msg));
         assertEquals(msg, decrptedMsg);
+
+        try {
+            StringHelper.decrypt("ABC");
+            signalFailureToDetectException("Decryption of invalid ciphertext should throw");
+        } catch (InvalidParametersException e) {
+            ignoreExpectedException();
+        }
```

Comment:
Perhaps use ExpectedException technique here?
http://howtodoinjava.com/testng/testng-expected-exception-and-expected-message-tutorial/